src/shared/services/map-location/index.js

```diff
@@ -1,28 +1,60 @@
-function mapLocation(loc) {
+export const location2feature = location => ({
+  type: 'Point',
+  coordinates: [location.lng, location.lat],
+});
+
+export const feature2location = feature => {
+  const { coordinates } = feature;
+  return {
+    lat: coordinates[1],
+    lng: coordinates[0],
+  };
+};
+
+export const address2pdok = address => {
```

Comment:
This function doesn't seem to be used. Please remove if not used.

Reply:
This one is used in the mapLocation conversion.